electron/src/runtime/lifecycle.ts

```diff
@@ -83,7 +83,7 @@ export const addRelaunchListeners = (listener: () => void) => {
   relaunchListeners.push(listener);
 };
 
-export const quit = async (clearCache = true): Promise<void> => {
+export const quit = async (clearCache = false): Promise<void> => {
```

Comment:
So I'm guessing in most cases we were actually clearing the cache because there was not arguments, right?

Reply:
Correct, there was only one case where we'd not clear it